backend/serializers/participant_serializer.py

```diff
@@ -3,14 +3,36 @@
 
 class ParticipantSchema(Schema):
     class Meta:
-        fields = ('name', 'lastname', 'email', 'github', 'phone')
+        fields = (
+            'id', 'name', 'lastname', 'email', 'github', 'hacknights', 'phone'
+        )
+        dump_only = ('id', 'hacknights')
 
     name = fields.Str(required=True, validate=[validate.Length(max=50)])
     lastname = fields.Str(validate=[validate.Length(max=50)])
     email = fields.Email(validate=[validate.Length(max=200)])
     github = fields.Str(validate=[validate.Length(max=200)])
+    hacknights = fields.Nested(
+        'HacknightSchema',
+        exclude=('participants', ),
```

Comment:
That's just one parameter, comma could be removed?

Reply:
That is tuple, and comma is required. We could pass list instead of tuple, where comma is not required, but tuple is more common in marshmallow docs.